src/main/java/techcourse/myblog/article/Article.java

```diff
@@ -0,0 +1,78 @@
+package techcourse.myblog.article;
```

Comment:
계층 우선 패키지 구조와 기능 우선 패키지 구조가 혼재되어 있어요. 아래의 글을 읽고 계층 우선 패키지 구조로 작성해 보세요.
https://www.slipp.net/questions/36

```suggestion
package techcourse.myblog.domain.article;
```

Reply:
네! 수정했습니다. 좋은 문서 감사합니다~~!